package.json

```diff
@@ -19,7 +19,7 @@
     "jest": "^26.0.0",
     "mocha": "^8.0.1",
     "nixt": "github:vesln/nixt#d2075099ecf935c870705fa5d5de437113d4d272",
-    "origami-build-tools": "^10.5.9",
+    "origami-build-tools": "prerelease",
```

Comment:
```suggestion
    "origami-build-tools": "^11",
```

Reply:
fixed in [`7733640` (#238)](https://github.com/Financial-Times/create-origami-component/pull/238/commits/7733640bb2ff9c761fd2babfa3fe5e8260d1db84)